tests/unit/models/builder-test.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
-var fs              = require('fs-extra');
+var fs              = require('fs');
+var fse             = require('fs-extra');
```

Comment:
check the assert in the test

We want to assert that we have modified `fs.statSync`.  Because `fse.statSync !== fs.statSync` we can't trivially do this via `fs-extra`